src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownyEventListener.java

```diff
@@ -180,4 +182,12 @@ public void onOutlawTeleportEvent(OutlawTeleportEvent event) {
     	if (SiegeWarSettings.getWarSiegeEnabled() && SiegeController.hasActiveSiege(event.getTown())) 
     		event.setCancelled(true);
     }
+
+    @EventHandler(ignoreCancelled = true)
+    public void on(NationToggleNeutralEvent event) {
+        if (event.getFutureState()) {
+            event.setCancelled(true);
+            event.setCancelMessage(Translation.of("msg_err_nation_neutrality_not_supported"));
+        }
+    }

```

Comment:
This ought to be in the SiegeWarNationEventListener class.

Reply:
Fixed